### PR TITLE
keybindings api

### PR DESCRIPTION
**Pull request by noskill**
impements attack use and selecting hotbar items

should fix https://github.com/noskill/Vereya/issues/2

### CODE REVIEW

File: src/main/java/io/singularitynet/MissionHandlers/CommandForAttackAndUseImplementation.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+package io.singularitynet.MissionHandlers;
+
+public class CommandForAttackAndUseImplementation extends CommandGroup
+{
+    public CommandForAttackAndUseImplementation()
+    {
+        super();
+        setShareParametersWithChildren(true);	// Pass our parameter block on to the following children:
+        addCommandHandler(new CommandForKey("key.attack"));
+        addCommandHandler(new CommandForKey("key.use"));
```
[the remark this answers is not in the document]
Comment (CICS-Oleg) [reply src/main/java/io/singularitynet/MissionHandlers/CommandForAttackAndUseImplementation.java:29]: I've tried to fetch this pr, build and test but the first try to send command "use" return nothing... After rebuilding project errors occurred.

[the remark this answers is not in the document]
Comment (noskill) [reply src/main/java/io/singularitynet/MissionHandlers/CommandForAttackAndUseImplementation.java:29]: CommandForKey("key.use") implements command for use/place block . It's right mouse click by default

[the remark this answers is not in the document]
Comment (noskill) [reply src/main/java/io/singularitynet/MissionHandlers/CommandForAttackAndUseImplementation.java:29]: I also got " Could not retrieve declared methods org.eclipse.jdt.internal.compiler.problem.AbortCompilation: Pb(324)" error

https://bugs.eclipse.org/bugs/show_bug.cgi?id=446577 it pops up occasionally

File: build.gradle [hunk 2]
```diff
@@ -112,4 +112,17 @@ classes {
...
+gradle.taskGraph.whenReady {taskGraph ->
+	println "Found task graph: " + taskGraph
+	println "Found " + taskGraph.allTasks.size() + " tasks."
+	taskGraph.allTasks.forEach { task ->
+		println task
+		task.dependsOn.forEach { dep ->
+			println "  - " + dep
+		}
+	}
+}
```
Comment (CICS-Oleg) [inline build.gradle:126]: Is it for debug print only?

**Review by CICS-Oleg — APPROVED** [review]
Is it for debug print only

File: build.gradle (hunk 2 above)
Comment (noskill) [reply build.gradle:126]: yes

**Pull request merged**